prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts

```diff
@@ -6,6 +6,7 @@ const COMMUNITY_PHARMACY_ORGANIZATION_AGREEMENT = 'Community Pharmacy Organizati
 const PHARMACY_OBO_TOA = 'Pharmacy On Behalf Of User';
 const DEVICE_PROVIDER_ORG_AGREEMENT = 'Device Provider Organization';
 const PHARMACY_TECHNICIAN_TOA = 'Pharmacy Technician Toa';
+const LICENCED_PRACTICAL_NURSE_TOA = 'Licenced Practical Nurse';
```

Comment:
For clarity, should that be `Licenced Practical Nurse Toa`

Reply:
it is fixed.